nservicebus/pipeline/steps-stages-connectors_extra_core_[6.0,).partial.md

```diff
@@ -14,6 +14,7 @@ In the diagram User Code can refer to a handler or a saga. If the handler or sag
 
 ### Incoming Pipeline Stages
 
+ * TransportReceiveContext: Behaviors in this stage are responsible signaling failure or success to the transport. This stage provides `ITransportReceiveContext` to it's behaviors.
```

Comment:
@weralabaj I had a call with @ramonsmits he should have all the context to update the wording. Since you asked me what about:

Behaviors in this stage are responsible signalling failure or success to the transport. In this stage no outgoing operations are supported over the context since the outbox transaction and the batching starts right before the physical message context. This stage provides `ITransportReceiveContext` to it's behaviors.

or to verbose?